### PR TITLE
Add expose? option to interfaces

### PR DESCRIPTION
**Pull request by leoiacovini**
When we don't want to create ingresses for a given interface we can mark it as `:expose? false`

### CODE REVIEW

File: src/soil/adapters/application.clj [hunk 1]
```diff
@@ -98,26 +98,27 @@
...
-                       {:ports          ports
-                        :tcp-interfaces tcp-interfaces})))))
+                              {:ports          ports
+                               :tcp-interfaces tcp-interfaces})))))
 
 (s/defn application->urls :- schemas.application/ApplicationUrls
   [application :- models.application/Application]
   (->> (logic.application/get-non-tcp-interfaces application)
-       (mapv (fn [{:interface/keys [name host]}] {name host}))
+       (mapv (fn [{:interface/keys [name host type]}] {name (str (name type) "://" host)}))
```
Comment (leoiacovini) [inline src/soil/adapters/application.clj]: Also I am adding the protocol when rendering the Application links

File: src/soil/schemas/application.clj [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
                                                   :image                      s/Str
                                                   (s/optional-key :syncable?) s/Bool
                                                   :env                        (s/pred map?)}]
-                                    :interfaces [{:name      s/Str
-                                                  :port      s/Int
-                                                  :container s/Str
-                                                  :type      InterfaceType}]})
+                                    :interfaces [{:name                     s/Str
+                                                  :port                     s/Int
+                                                  (s/optional-key :expose?) s/Bool
```
Comment (RafaeLeal) [inline src/soil/schemas/application.clj]: If you're going to add this, we probably want to change the adapter `application->ingress` to filter by this property

File: src/soil/adapters/application.clj (hunk 1 above)
Comment (RafaeLeal) [reply src/soil/adapters/application.clj]: :see_no_evil:

File: build.sh [hunk 3]
```diff
@@ -2,11 +2,13 @@
 
 set -eou pipefail
 
+version=$(cat ./resources/base.edn | grep  ':version .*"[0-9\.]*"' | grep -oEi '\d*\.\d*\.\d*')
```
Comment (RafaeLeal) [inline build.sh]: dat bash :1st_place_medal:

File: src/soil/schemas/application.clj (hunk 2 above)
Comment (leoiacovini) [reply src/soil/schemas/application.clj]: https://github.com/formicarium/soil/pull/17/files#diff-52f04ec18cc774622c53aa0e9fbb2c54R102

**Pull request merged**